slides/src/content/introduction.md

```diff
@@ -54,3 +54,280 @@
+
+---
+
+## First Look at the Component
+
+_app.component.ts_
+```ts
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  template: '<h1>{{title}}</h1>',
+  styles: ['h1 { color: font-size: 30px; }']
+})
+export class AppComponent {
+  title = 'app works!';
+}
+```
+
+- `import` statement gives you access to Angular components
+- `@Component` is a decorator that tells Angular this class is a component
+- `selector` is the way you intend to reference the component
+  - Used like an HTML tag `<app-root></app-root>`
+- `template` is what gets rendered onto the page
+  - For web applications, this is HTML
+- `styles` are CSS stylesheets that get applied to your component
+- `AppComponent` is a class that decides how the component should act
+
+---
+
+## What is a Component?
+
+- The core building block of an Angular application is the component
+- In Angular, a component is some chunk of application logic combined with something that gets displayed
+- Everything being displayed on any page should be associated with some component
+
+---
+
+## Component Template Syntax
+
+- `<h1>{{title}}</h1>` represents __interpolation__
+- Interpolation attempts to evaluate the expression between the curly braces and turn that into a string
+- Symbols (such as title) are evaluated within an expression context
+  - in this case, the component's class members
+
+---
+
+## Generating a New Component
+
+- We'll use Angular CLI to create a new component
+- `ng generate component [component path]`
+- Component gets added to the list of declarations
+- Generated component follows the same structure as `AppComponent`
+- Prefixes `app` on both components. This is an Angular best practice
+- Running `ng generate component feature` generates the following file structure:
+```
+├── src
+│   ├── app
+│   │   ├── app.component.css
+│   │   ├── app.component.html
+│   │   ├── app.component.spec.ts
+│   │   ├── app.component.ts
+│   │   ├── app.module.ts
+│   │   └── feature
+│   │       ├── feature.component.css
+│   │       ├── feature.component.html
+│   │       ├── feature.component.spec.ts
+│   │       └── feature.component.ts
+│   ├── assets
+│   ├── environments
+│   │   ├── environment.prod.ts
+│   │   └── environment.ts
+│   ├── favicon.ico
+```
+
+---
+
+## Integrating a Component
+
+Every component has a selector:
+
+*example.component.ts*
+```typescript
+@Component({
+	selector: 'app-example',
+	template: '<p>Multiple component example</p>'
+})
+export class ExampleComponent { }
+```
+
+---
+
+## Integrating a Component
+
+Use selector as an HTML tag within another component's template
+
+
+*app.component.ts*
+```typescript
+@Component({
+	selector: 'app-root',
+	template: `
+	  <h1>My Angular App</h1>
+	  <app-example></app-example>
+	`
+})
+export class AppComponent { }
+```
+[View Example](https://plnkr.co/edit/CMGx2TLUdPcL0D4tvGI9?p=preview)
+
+---
+
+## What is a directive?
```

Comment:
@gvwilson @jamesbs This was my proposed slide addition - would appreciate your feedback on this (I know James, you felt it more appropriate to give just enough information to get them to `*ngIf`)